### ISSUE TITLE
Feature: Support ISO timestamps for period selector

### ISSUE THREAD

**Issue by jason-p-pickering** [pepfar-datim/datim-dedupe#65]
```
  "RESULTS": {
    "2017Q2": {
      "start": 1498885200,
      "end": 1506805200,
      "datasets": [
      ....
      ]
    }
  }
```

provides the current datastore configuration. 

Swtich to 

```
  "RESULTS": {
    "2017Q2": {
       "start": "Jul 01 2017 01:00:00 GMT-0500"
      "end": "Sep 30 2017 17:00:00 GMT-0500",
      "name": "April - June 2017",
      "datasets": [
      ....
      ]
    }
  }
```

This will make this more consistent with the approval app and allow for easier updating with the datastore app. 

Will require modification of both the client and server side function. 

@Markionium FYI.

**Comment by jason-p-pickering** [comment]
@Markionium I have added support for ISO formatted timestamps such as 

```
      "start": "2016-07-14T05:06:43+00:00",
      "end": "2018-07-14T05:06:43+00:00",
```

in the Postgres function. Could you update the app to support these as well? 

The function will accept either Epoch timestamps or ISO timestamps, but I think it would also be OK to support only ISO timestamps.

**Comment by MDHomem** [comment]
Fixed in #78
